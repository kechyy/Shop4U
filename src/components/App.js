import React from 'react';

const App = () => {
  return (
    <h1>Welcome to Shop4U</h1>
  )
}
export default App;