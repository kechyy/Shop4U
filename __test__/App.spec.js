import React from 'react';
import { configure, mount, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import toJson from 'enzyme-to-json';
import App from '../src/components/App';

configure({ adapter: new Adapter() });



describe('Sample Test', () => {
  it('Should return a successful sample test', () => {
    expect(true).toBeTruthy();
})
  it('Should render App without crashing', () => {
  const Wrapper = mount(<App />);

    expect(toJson(Wrapper)).toMatchSnapshot();
  })

  it('should render home without crashing', () => {
  const Wrapper = mount(<App />);
    expect(Wrapper).toBeDefined();
    expect(toJson(Wrapper)).toMatchSnapshot();

  })
  it('should find h1 element', () => {
    const wrapper = mount(<App />);
    expect(wrapper.find('h1').length).toEqual(1);
    expect(wrapper.find('h1').text()).toBe('Welcome to Shop4U');
  });
})