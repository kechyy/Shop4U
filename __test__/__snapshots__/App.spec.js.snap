// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sample Test Should render App without crashing 1`] = `
<App>
  <h1>
    Welcome to Shop4U
  </h1>
</App>
`;

exports[`Sample Test should render home without crashing 1`] = `
<App>
  <h1>
    Welcome to Shop4U
  </h1>
</App>
`;
